app/src/main/res/layout/view_description_edit.xml

```diff
@@ -180,6 +180,23 @@
                         </com.google.android.material.textfield.TextInputLayout>
                     </FrameLayout>
 
+                    <com.google.android.material.button.MaterialButton
+                        android:id="@+id/learnMoreButton"
+                        style="@style/FlatButton.Small.Transparent"
+                        android:layout_width="match_parent"
+                        android:layout_marginStart="12dp"
```

Comment:
Can we make the start and the end margins balanced? Only applying `marginStart` will make the ripple animation look aligned to the right instead of to the middle.
<img src="https://user-images.githubusercontent.com/2435576/189244320-524de359-0323-4aeb-9e8e-e5cebe290ddb.jpg" width="480" />

Reply:
Done @cooltey However, I feel just matching the margin numbers might not resolve the issue on all device sizes. It definitely looks better.